source/index.rst

```diff
@@ -24,4 +24,5 @@ Welcome to |LPN| (version |version|) documentation!
    security/index
    how-tos/index
    tools/index
+   tutorial/index
```

Comment:
It'd be okay for now, but recommend to have separate commits or pull requests for different topic.